src/dht/socket.rs

```diff
@@ -187,16 +187,13 @@ impl DHTSocket {
 
                 match request_info {
                     Some(request_info) => {
-                        match request_info.response_channel {
-                            Some(response_channel) => {
-                                if let Err(e) = response_channel.send(message.clone()).await {
-                                    let message = e.0;
-                                    warn!(target: "rustydht_lib::DHTSocket", "Got response, but sending code abandoned the channel receiver. So sad. Response: {:?}. Sender: {:?}", message, sender);
-                                }
-                            }
-                            None => { /*It's fine if there's no channel - just means that the sender didn't care about getting a response*/
+                        if let Some(response_channel) = request_info.response_channel {
+                            if let Err(e) = response_channel.send(message.clone()).await {
+                                let message = e.0;
+                                warn!(target: "rustydht_lib::DHTSocket", "Got response, but sending code abandoned the channel receiver. So sad. Response: {:?}. Sender: {:?}", message, sender);
                             }
                         }
+                        // It's fine if there's no channel - just means that the sender didn't care about getting a response
```

Comment:
Please move the comment to be before the `if let`
```suggestion
                        // It's fine if there's no channel - just means that the sender didn't care about getting a response
                        if let Some(response_channel) = request_info.response_channel {
                            if let Err(e) = response_channel.send(message.clone()).await {
                                let message = e.0;
                                warn!(target: "rustydht_lib::DHTSocket", "Got response, but sending code abandoned the channel receiver. So sad. Response: {:?}. Sender: {:?}", message, sender);
                            }
                        }
```

Reply:
Done